xesmf/frontend.py

```diff
@@ -925,6 +932,7 @@ def __init__(
         ----------
         This approach is inspired by `OCGIS <https://github.com/NCPP/ocgis>`_.
         """
+        # Note, I suggest we refactor polys -> geoms
```

Comment:
I have no objection.

Reply:
Ok, let's consider that in a future refactor.